e2e/config/config_test.go

```diff
@@ -254,3 +260,67 @@ func TestConfigNotEqual(t *testing.T) {
 		})
 	}
 }
+
+func TestValidateDeployers(t *testing.T) {
```

Comment:
This is the same way we test pvcSepcs, but it would be simpler if we split to TestValidDeployers and TestInvalidDeployers like this:

- https://github.com/RamenDR/ramenctl/blob/1f58857d9d5bb4e3c5a8cb2f670e64975243cbba/pkg/report/report_test.go#L333
- https://github.com/RamenDR/ramenctl/blob/1f58857d9d5bb4e3c5a8cb2f670e64975243cbba/pkg/report/report_test.go#L350
B

But for now I think this is fine.